hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/service/DirectoryDeletingService.java

```diff
@@ -221,11 +225,11 @@ public BackgroundTaskResult call() {
           long dirNum = 0L;
           long subDirNum = 0L;
           long subFileNum = 0L;
-          long remainNum = pathLimitPerTask;
```

Comment:
we can remove pathLimitPerTask and related configuration - ozone.path.deleting.limit.per.task and default value.

Reply:
Done.